schemas/testpilot.js

```diff
@@ -1,21 +1,40 @@
 "use strict";
 
-const Joi = require("joi");
+const Joi = require("joi-browser");
 const commonSchema = require("./commonSchema");
 
-const testpilotEvent = Joi.object().keys({
-  event: Joi.string().required(),
-  object: Joi.string().required(),
-  timestamp: Joi.number().integer().required()
-});
-
+// commonSchema keys not shown below:
+// - event_type, called "event" in testpilot docs, and "method" in
+//   testpilot-metrics docs;
+// - client_id, called "clientId" in the Telemetry ping.
 const schema = commonSchema.keys({
-  payload: Joi.object().required().keys({
-    events: Joi.array().required().length(1).items(testpilotEvent),
-    test: Joi.string().valid("@testpilot-addon").required(),
-    timestamp: Joi.number().integer().required(),
-    version: Joi.string().required()
-  })
-}).options({allowUnknown: true});
+  // Require that the commonSchema "topic" field always be "testpilot".
+  topic: Joi.string().valid("testpilot").required(),
+
+  // Additional "testpilot" event fields:
+  // object: what was affected by "event", e.g., a button.
+  object: Joi.string(),
+
+  // client_time: non-PII timestamp, integer number of seconds since Firefox
+  //              application start.
+  client_time: Joi.number().integer().required(),
+  // variants: an optional string (either a simple string, or serialized JSON)
+  //           that describes any A/B or multivariate tests currently running.
+  //           See the variant docs in the mozilla/testpilot repo for more.
+  variants: Joi.string(),
```

Comment:
I looked this up in the test pilot repo:
```
variants: [
      {
        value: "a",
        weight: 1,
        description: "Option A."
      },
      {
        value: "b",
        weight: 1,
        description: "Option B."
      }
    ]
```

So like you say in the comment this describes the variants currently running? In my mind I imagined this would be a label of which variant this data belongs to. That way, again, graphs can be generated easily to compare variants. So it would just say `"variant": "a"` or `"variant": "b"`

Would that make sense here? Is there another field that says what variant the data belongs to? Or am I misunderstanding something?